engines/sci/detection.cpp

```diff
@@ -85,11 +85,11 @@ static const PlainGameDescriptor s_sciGameTitles[] = {
 	{"lsl3",            "Leisure Suit Larry 3: Passionate Patti in Pursuit of the Pulsating Pectorals"},
 	{"mothergoose",     "Mixed-Up Mother Goose"},
 	{"pq2",             "Police Quest II: The Vengeance"},
-	{"qfg1",            "Quest for Glory I: So You Want to Be a Hero"},	// Note: There was also a SCI11 VGA remake of this (further down)
+	{"qfg1",            "Hero's Quest: So You Want to Be a Hero"},	// Note: There was also a SCI11 VGA remake of this (further down) called Quest for Glory I: So You Want to Be a Hero
 	{"sq3",             "Space Quest III: The Pirates of Pestulon"},
 	// === SCI01 games ========================================================
 	{"qfg2",            "Quest for Glory II: Trial by Fire"},
-	{"kq1sci",          "King's Quest I: Quest for the Crown"},	// Note: There was also an AGI version of this
+	{"kq1sci",          "Roberta Williams' King's Quest I: Quest for the Crown"},	// Note: There was also an AGI version of this called King's Quest: Quest for the Crown
```

Comment:
While this is theoretically correct, I'm not sure it's a great idea. If  we add both the agi and the sci versions,
in the launcher we get two entries with different names:
```
King's Quest: Quest for the Crown
Roberta Williams' King's Quest I: Quest for the Crown
```

Reply:
Is the problem with this one that when we add a game to the launcher it will bring both up? I assumed the Engines would each individually bring up the games they label themselves in their detection groups am I wrong about that? I guess what is the situation that this conflict happens? and if it does happen won't it be more conflict to have no difference at all because because then it would just be doubles of the same title that pops up with no way to tell the engine diffrence?

Reply:
Ill leave this one unedited because I could see an argument for both ways

Reply:
> Is the problem with this one that when we add a game to the launcher it will bring both up?
No, it's just a matter of usability. The gamelist is sorted by game name, if we add the "Roberta Williams" string, the SCI entry would fall down among the "R" entries, although it's the same game as the AGI version. Not a big issue, but I think we should strike a balance between being faithful to the original and easy to use from a player's point of view.

Reply:
Yes, this is the only one I don't like. Hero quest name being different than the sci version is kind of iconic because of the lawsuit that sierra were hit with to making them change the name and it's in the intro, but 'Roberta William's' feels like 'Author presents .... title' to me, not a 'real' part of title.

Reply:
I agree with @tag2015 here. It's best to drop the "Roberta Williams" prefix for uniformity and ease of use

```suggestion
	{"kq1sci",          "King's Quest I: Quest for the Crown"},	// Note: There was also an AGI version of this called King's Quest: Quest for the Crown
```